__tests__/app.test.js

```diff
@@ -82,4 +82,66 @@ describe('/api/articles/:id', () => {
             expect(err.response.body.msg).toEqual("Article does not exist")
         })
     })
+})
+describe('/api/articles', () => {
+    test('responds with an array of article objects with comment_count created from comments.js', () => {
+        return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then((response) => {
+            const articles = response.body.articles
+            expect(articles).toBeInstanceOf(Array);
+            expect(articles.length).toEqual(13)
+            articles.forEach((article) => {
+                expect(article).toHaveProperty('author');
+                expect(article).toHaveProperty('title');
+                expect(article).toHaveProperty('article_id');
+                expect(article).toHaveProperty('topic');
+                expect(article).toHaveProperty('created_at');
+                expect(article).toHaveProperty('votes');
+                expect(article).toHaveProperty('article_img_url');
+                expect(article).toHaveProperty('comment_count');
+        })
+        })
+    })
+    test('comment count is accurate', () => {
+        return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then((response) => {
+            const articles = response.body.articles;
+            expect(articles[0].comment_count).toEqual(2);
+            expect(articles[1].comment_count).toEqual(1);
+            expect(articles[2].comment_count).toEqual(0)
+        })
+    })
+    test('body property is not present', () => {
```

Comment:
You could combine the test checking what properties are there with an assertion that this property is missing to make it more concise.